package.json

```diff
@@ -56,6 +56,7 @@
     "react-refresh": "0.10.0",
     "tailwindcss": "3.2.7",
     "typescript": "4.9.5",
-    "url-loader": "4.1.1"
+    "url-loader": "4.1.1",
+    "zod": "3.21.4"
```

Comment:
zod is great for validation with typescript. We can use this to ensure that the HTTP request body matches the typescript type before continuing with the request logic